test/testutils/util.go

```diff
@@ -472,3 +472,17 @@ func DeleteNamespace(ctx context.Context, c client.Client, ns *corev1.Namespace)
 	}
 	return nil
 }
+
+func SetLeaderPodGroupToReady(ctx context.Context, k8sClient client.Client, lws *leaderworkerset.LeaderWorkerSet, podGroupNumber int32) {
```

Comment:
I think what you need is `SetLeaderPodsToReady` rather than the groups because worker statefulsets are not created.